### DISCUSSION TITLE
Any chance for functionality "moduleFileExtensions"?

### DISCUSSION THREAD

**Discussion by alexbot89** [vitest-dev/vitest#2796] [Q&A] [answered]
I've run into an issue where if I omit the `.vue` extension on my imports like so `./mycomponent` vitest will not pick it up even though it exists.

I can do the same thing in Jest via `moduleFileExtensions` where I can omit the extensions and have it automatically detected.

https://jestjs.io/docs/configuration#modulefileextensions-arraystring

Is there anything similar I can do with vitest?

**Comment by AriPerkkio** [comment, marked as answer]
Does it work with `resolve.extensions` option?

https://vitejs.dev/config/shared-options.html#resolve-extensions

```ts
import { defineConfig } from "vitest/config";

export default defineConfig({
  resolve: {
    extensions: [".vue"],
  },
  ...
```

**Reply by alexbot89** [reply]
Yes, thank you, this is exactly what I was looking for, adding both `['.js', '.vue']` did the trick.

The project I am working on is only using vitest as the runner and I did not quite find in the vitest documentation that this could be done in the `vitest.config.ts`.
